### PR TITLE
allow to set COCKROACH_ENGINE_MAX_SYNC_DURATION

### PR DESCRIPTION
**Pull request by christianhuening**
Signed-off-by: Christian Hüning <christian.huening@figo.io>

#### What this PR does / why we need it:
we need it because CRDB support told us to set COCKROACH_ENGINE_MAX_SYNC_DURATION but we couldn't via the helm chart
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Review by a-robinson — COMMENTED** [review]
Hi @christianhuening, thanks for the contribution. I suspect that we'd be better off adding a `ExtraEnvVars` parameter which can be used to set arbitrary environment variables in the config. Would you be up for changing to that?

**Comment by christianhuening** [comment]
@a-robinson Agreed, I switched to an ExtraEnvArgs option. Also I bumped the default tag to 2.1.6 and increased the Chart "App Version" accordingly.

File: stable/cockroachdb/values.yaml [hunk 1]
```diff
@@ -72,3 +72,5 @@ Locality: ""
 ExtraArgs: []
 # ExtraSecretMounts is a list of names from secrets in the same namespace as the cockroachdb cluster, which shall be mounted into /etc/cockroach/secrets/ for every cluster member.
 ExtraSecretMounts: []
+# ExtraEnvArgs is a list of name,value tuples providing extra ENV variables
```
Comment (a-robinson) [inline stable/cockroachdb/values.yaml]: It's not clear to me from this comment how I should format the environment variables into an array, given that each has a name and a value. Mind clarifying it with an example or something?

**Review by a-robinson — COMMENTED** [review]
Thanks for the update! Just one small request, then this should be good to go.

File: stable/cockroachdb/values.yaml (hunk 1 above)
Comment (christianhuening) [reply stable/cockroachdb/values.yaml]: done

**Comment by a-robinson** [comment]
/ok-to-test
/lgtm
/approve

Thanks!

**Pull request merged**